packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -1,143 +1,64 @@
-%if 0%{?rhel} == 7 || (0%{?fedora} && 0%{?fedora} <= 29)
-%bcond_without python2
-%else
-%bcond_with python2
-%endif
-
-%if 0%{?rhel} >= 8 || 0%{?fedora} || 0%{?epel}
-%bcond_without python3
-%else
-%bcond_with python3
-%endif
-
 # template: smart_proxy_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
-%{!?_root_bindir:%global _root_bindir %{_bindir}}
-%{!?_root_sbindir:%global _root_sbindir %{_sbindir}}
-%{!?_root_datadir:%global _root_datadir %{_datadir}}
-%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
-%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
-
 %global gem_name smart_proxy_salt
 %global plugin_name salt
 
-%global foreman_proxy_min_version 2.5
-%global foreman_proxy_dir %{_root_datadir}/foreman-proxy
-%global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
+%global foreman_proxy_min_version 1.24
+%global foreman_proxy_dir %{_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_sharedstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
-%global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
-
-%global salt_config_dir %{_root_sysconfdir}/salt
-%global salt_proxy_runners_dir %{foreman_proxy_dir}/salt/runners
-%global salt_proxy_reactors_dir %{foreman_proxy_dir}/salt/reactors
-%global salt_state_grains_dir %{foreman_proxy_statedir}/salt/grains
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
 
-Summary: SaltStack support for Foreman Smart-Proxy
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 5.0.1
+Name: rubygem-%{gem_name}
+Version: 5.1.0
 Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
+Summary: SaltStack Plug-In for Foreman's Smart Proxy
 License: GPLv3
 URL: https://github.com/theforeman/smart_proxy_salt
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: salt-master
-%if %{with python2}
-Requires: python
-BuildRequires: python2-rpm-macros
-%endif
-%if %{with python3}
-Requires: python3
-BuildRequires: python3-rpm-macros
-%endif
-
 # start specfile generated dependencies
 Requires: foreman-proxy >= %{foreman_proxy_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(smart_proxy_dynflow) >= 0.5.0
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.7
```

Comment:
I think we should set this to >= 3 (have a look how foreman does this)

Reply:
I had a look at other plugins and saw either no version limit, or 2.5 or 2.7.... Looks to me like this is okay